src/test/resources/events/events.json

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "@context": "https://bluebrain.github.io/nexus/contexts/resource.json",
+    "@type": "Created",
+    "resourceId": "http://example.com/base/Created",
```

Comment:
I'm wondering if we should use `@id` here or not; for realms we do use `@id`; we don't for orgs and projects as they define labels and uuids instead. Thoughts?

Reply:
I think @id in the events does not make sense if we follow the idea that an event is NOT the resource itself. But well It wouldn't be tragic if we would use it either...